adage/analyze/api.py

```diff
@@ -275,12 +275,37 @@ class Meta:
         queryset = Node.objects.all()
         resource_name = 'node'
         allowed_methods = ['get']
+        multiple_allowed_methods = ['post']
```

Comment:
Yes. This follows the technique we developed a while back for taking advantage of the built-in Tastypie `dispatch` functionality.